src/test/scala/uk/gov/nationalarchives/tdr/api/service/TransferAgreementServiceSpec.scala

```diff
@@ -61,18 +60,10 @@ class TransferAgreementServiceSpec extends AnyFlatSpec with MockitoSugar with Ma
       userId
     ).futureValue
 
-    val consignmentStatusResult: Seq[ConsignmentstatusRow] =
-      consignmentStatusRepositoryMock.getConsignmentStatus(consignmentId).futureValue
-
     transferAgreementResult.consignmentId shouldBe consignmentId
     transferAgreementResult.allCrownCopyright shouldBe true
     transferAgreementResult.allEnglish shouldBe true
     transferAgreementResult.allPublicRecords shouldBe true
-
-    consignmentStatusResult.head.consignmentid shouldBe consignmentId
-    consignmentStatusResult.head.consignmentstatusid shouldBe consignmentStatusId
-    consignmentStatusResult.head.statustype shouldBe statusType
-    consignmentStatusResult.head.value shouldBe statusValue
   }
 
   "addTransferAgreementCompliance" should "add the correct metadata given correct arguments and set TA status to Completed" in {
```

Comment:
Not sure this test is actually checking that the status has been set to `Completed`

Reply:
Ah, yes, you are correct, I was meant to remove this, but now that you mentioned the `ArgumentCaptor`, I might use that instead.